sycl/include/CL/sycl/intel/sub_group.hpp

```diff
@@ -280,18 +280,26 @@ struct sub_group {
 
   template <typename T, access::address_space Space>
   typename std::enable_if<(sizeof(T) == sizeof(uint32_t) ||
-                           sizeof(T) == sizeof(uint16_t)) &&
-                              Space == access::address_space::global_space,
+                           sizeof(T) == sizeof(uint16_t) ||
+                           sizeof(T) == sizeof(uint8_t)) &&
+                          Space == access::address_space::global_space,
                           T>::type
   load(const multi_ptr<T, Space> src) const {
     if (sizeof(T) == sizeof(uint32_t)) {
       uint32_t t = __spirv_SubgroupBlockReadINTEL<uint32_t>(
           (const __attribute__((ocl_global)) uint32_t *)src.get());
       return *((T *)(&t));
     }
-    uint16_t t = __spirv_SubgroupBlockReadINTEL<uint16_t>(
-        (const __attribute__((ocl_global)) uint16_t *)src.get());
-    return *((T *)(&t));
+    else if (sizeof(T) == sizeof(uint16_t)) {
```

Comment:
Please, do not use `else` after `return`.
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Also, apply clang-format to fix formatting.